pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -1454,10 +1450,6 @@ public ScheduledExecutorService getExecutor() {
         return executor;
     }
 
-    public ScheduledExecutorService getCacheExecutor() {
-        return cacheExecutor;
-    }
```

Comment:
This is technically a breaking change for pulsar extensions. I don't know of any extensions using it, but it seems risky to just remove the method. @BewareMyPower - what is your opinion on the right way to handle these kinds of methods that could be used by extensions?